### DISCUSSION TITLE
Delay in Automation from Physical Device

### DISCUSSION THREAD

**Discussion by djqpaz** [SecKatie/ha-wyzeapi#499] [Q&A]
Is there a way to change the update time for Wyze switches to Home assistant?  

I have the wyze switches trigger other devices when they are turned on or off, but there is about a 20-30 second delay if the physical switch is pressed. 

i looked through the past discussion and issue and didn't see anything that could explain it or fix it. 

any Ideas out there?

**Comment by SecKatie** [comment]
The big issue is that we can’t poll very quickly due to rate limits if we let it be configured to be shorter then they said they will block us

On Sat, Jul 29, 2023, at 1:01 PM, djqpaz wrote:
> 
> 
> Is there a way to change the update time for Wyze switches to Home assistant?
> 
> I have the wyze switches trigger other devices when they are turned on or off, but there is about a 20-30 second delay if the physical switch is pressed.
> 
> i looked through the past discussion and issue and didn't see anything that could explain it or fix it.
> 
> any Ideas out there?
> 
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/SecKatie/ha-wyzeapi/discussions/499>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AIU2G3ZLD5IGUPKG42GQXRLXSU6XHANCNFSM6AAAAAA24TPHXQ>.
> You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
> 
--
Katie Mulliken ***@***.***>
